401code_challenges/codeChallenge/src/main/java/fizzBuzzTree/App.java

```diff
@@ -0,0 +1,48 @@
+package fizzBuzzTree;
+
+public class App {
+
+
+    public static void main(String[] args){
+        Tree tr = new Tree();
+        tr.root=new Node(5);
+        tr.root.leftChild=new Node(15);
+        tr.root.rightChild=new Node(4);
+        tr.root.leftChild.leftChild=new Node(9);
+        Object[] r = tr.preOrder();
+        for(Object o:r){
+            System.out.println(o);
+        }
+          Tree res = fizzBuzzTree(tr.root,tr);
+
+System.out.println("++++++++++++++++++++++++++");
+        Object[] rr = res.preOrder();
+        for(Object o:rr){
+            System.out.println(o);
+        }
+    }
+
+
+    public static Tree fizzBuzzTree(Node node,Tree tr) {
```

Comment:
I don't think it's necessary to take in a node and tree, I bet you could have figured it out using just one